platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/ConfigurationAdmin.java

```diff
@@ -374,6 +378,33 @@ public void setBundleLocation(String pid, String location) throws IOException {
         config.setBundleLocation(location);
     }
 
+    public boolean updateGuestClaimsProfile(String pid, Map<String, Object> configurationTable)
+            throws IOException {
+        try {
+            Object profileObj = configurationTable.get(PROFILE_KEY);
+            if (profileObj == null) {
+                return false;
+            }
+
+            String profile = (String) profileObj;
+            guestClaimsHandlerExt.setSelectedClaimsProfileName(profile);
+
+            Map<String, Object> currentUiConfig = getProperties(UI_CONFIG_PID);
+            Map<String, Object> defaultBannerMarkings =
+                    guestClaimsHandlerExt.getSelectedClaimsProfileBannerConfigs();
+
+            if (defaultBannerMarkings != null) {
+                currentUiConfig.putAll(defaultBannerMarkings);
+                update(UI_CONFIG_PID, currentUiConfig);
+            }
+            return true;
+        } catch (ClassCastException | IllegalArgumentException e) {
+            LOGGER.error("An invalid guest claims profile was selected, caused by: {}",
```

Comment:
I'm a little confused. What I was proposing is to do `LOGGER.debug("An invalid guest claims profile was selected", e)` instead of only logging the exception's message. That should give _more_ context, not less, right?